server/src/main/java/org/opensearch/extensions/ExtensionsManager.java

```diff
@@ -443,6 +447,15 @@ TransportResponse handleExtensionRequest(ExtensionRequest extensionRequest) thro
         }
     }
 
+    /**
+     * A helper method called during initialization that issues a service accounts to extensions
+     * @param extension The extension to be issued a service account
+     */
+    public String issueServiceAccount(DiscoveryExtensionNode extension) {
```

Comment:
Remove public modifier, external classes shouldn't use this IMO

Reply:
Good catch